test/integration/build-warnings/test/index.test.js

```diff
@@ -87,4 +87,9 @@ describe('Build warnings', () => {
     }))
     expect(stdout).not.toContain('no-cache')
   })
+
+  it('should warn when unknown argument provided', async () => {
+    const { stderr } = await nextBuild(appDir, ['--random'], { stderr: true })
+    expect(stderr).toEqual('Unknown or unexpected option: --random\n')
```

Comment:
Should this also check that there is no unhandled rejection warning?

Reply:
Thanks for pointing it out! a4b8b69